build-system/compile/post-closure-babel.js

```diff
@@ -101,8 +102,12 @@ async function postClosureBabel(file) {
     () => null,
     !argv.full_sourcemaps
   );
-
-  debug(CompilationLifecycles['complete'], file, compressed, sourceMap);
+  debug(
+    CompilationLifecycles['complete'],
+    file,
+    /** @type {string|undefined} */ (compressed),
```

Comment:
Is there a better way to do this? What is the original type of `compressed`? Can we just pass in `compressed?.toString()` here and ditch the cast?

Reply:
Yeah, that works